core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendConf.scala

```diff
@@ -35,7 +36,10 @@ trait ExternalBackendConf extends SharedBackendConf {
   def h2oClusterHost = sparkConf.getOption(PROP_EXTERNAL_CLUSTER_REPRESENTATIVE._1).map(_.split(":")(0))
   def h2oClusterPort = sparkConf.getOption(PROP_EXTERNAL_CLUSTER_REPRESENTATIVE._1).map(_.split(":")(1).toInt)
 
-  def numOfExternalH2ONodes = sparkConf.getOption(PROP_EXTERNAL_H2O_NODES._1)
+  @DeprecatedMethod("clusterSize")
+  def numOfExternalH2ONodes = clusterSize
+
+  def clusterSize = sparkConf.getOption(PROP_EXTERNAL_CLUSTER_SIZE._1)
```

Comment:
Shouldn't the method fall back to  `"spark.ext.h2o.external.cluster.num.h2o.nodes"` if `"spark.ext.h2o.external.cluster.size"` is not set?

Reply:
Yup, did that in train, but pushed now :)

Reply:
If this function is used only explicitly by an user or the original option is very unlikely to be specified then I think we don't have to stress about that. But if other part of SW utilize this function and a customer has the original option stored somewhere in some config (e.g. `spark-defaults.conf`), an upgrade to the next minor version of SW may cause unexpected problems.

Reply:
Good point, but I think we are still fine as we set the new value based on the deprecated value. I changed the code to centralize the logic. Now if you have the old option specified, and try to use H2OConf, you get warning
```
WARN H2OConf: Please use 'spark.ext.h2o.external.cluster.size' as 'spark.ext.h2o.external.cluster.num.h2o.nodes' is deprecated. Passing the value '1' to 'spark.ext.h2o.external.cluster.size'.
```

Reply:
Sounds good to me :-)